packages/@lwc/ssr-compiler/src/estree/validators.ts

```diff
@@ -26,9 +28,29 @@ export function isNullableOf<T extends Node>(validator: Checker<T>): NullableChe
 
 isNullableOf.__debugName = 'isNullableOf';
 
+/** A function that accepts a node and checks that it is a particular type of node. */
+export type Validator<T extends EsNode | null = EsNode | null> = {
+    (node: EsNode | null | undefined): node is T;
+    __debugName?: string;
+    __stack?: string;
+};
+
+const is: {
+    [K in keyof typeof _is]: (typeof _is)[K] & { __debugName?: string; __stack?: string };
+} = { ..._is };
+
 if (process.env.NODE_ENV !== 'production') {
-    // Modifying another package's exports is a code smell!
     for (const [key, val] of entries(is)) {
-        (val as any).__debugName = key;
+        val.__debugName = key;
```

Comment:
`val` here is still the function from `estree-toolkit`, so `is.identifier` imported via the actual package will still be modified. 😓

Reply:
True. I guess the only immediate win is making this compatible with esm, which admittedly isn't much for a debug-only feature.